src/Rings/mpoly-ideals.jl

```diff
@@ -2003,3 +2003,72 @@ function Base.show(io::IO, I::MPolyIdeal)
     print(io, "Ideal ", _get_generators_string_one_line(I))
   end
 end
+
+
+_pluecker_sgn(a::Vector{Int}, b::Vector{Int}, t::Int)::Int =
+  (-1)^(count(z -> z > t, a) + count(z -> z < t, b))
+
+@doc raw"""
+    flag_pluecker_ideal(dimensions::Vector{Int},n::Int,F)
```

Comment:
```suggestion
    flag_pluecker_ideal(dimensions::Vector{Int}, n::Int, F::Field)
```
type missing. or with the ordering change
```suggestion
    flag_pluecker_ideal(F::Field, dimensions::Vector{Int}, n::Int)
```